From: Ben Hutchings <ben@decadent.org.uk>
Date: Sat, 14 Jan 2012 14:07:30 +0000
Subject: [PATCH] firmware: Avoid ABI change in 2.6.32.17

Only request_firmware() should ever allocate struct firmware, so we
can safely add fields without callers knowing about them.

---
--- a/include/linux/firmware.h
+++ b/include/linux/firmware.h
@@ -11,7 +11,9 @@
 struct firmware {
 	size_t size;
 	const u8 *data;
+#ifndef __GENKSYMS__
 	struct page **pages;
+#endif
 };
 
 struct device;
